### DISCUSSION TITLE
Lookup Column values

### DISCUSSION THREAD

**Discussion by jerripothula** [microsoft-search/pnp-modern-search#3137] [Q&A]
version: 4.9.0.0
I am new to sharepoint. correct me if I'm wrong. Let me know if any information required.

I have document Library, which have documents in it.
I have created a pnp search results web part , search and filter for it.
It has lookup columns for customer , docType and Tags.
I have used the refinable string to map the crawled property.

For Example: See below screenshots
For the one of the documents shown below-
It is not showing Customer or tags, it showing only documentation.
For other Documents, it showing customer, not showing tags and doctype.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/46070806/1396bc45-2e08-4660-9ee6-f1dc8604f16f)
![image](https://github.com/microsoft-search/pnp-modern-search/assets/46070806/6fc74909-aedf-49ee-a790-de441b3f637b)
![image](https://github.com/microsoft-search/pnp-modern-search/assets/46070806/691f82bf-6869-425d-af97-53ec2e10cba6)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/46070806/8e3ec662-cddb-45a8-b1d8-8292b03515d3)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/46070806/cf151b0d-62a2-4eb7-aada-f0efdda1cd4b)

**Comment by kasperbolarsen** [comment]
First of all we will need to know how the field Customer is defined, like is it a list column or a site column.
Secondly can you share where you have mapped the crawled property to the RefinableString,  on the site collection or on tenant level?

**Reply by jerripothula** [reply]
Thanks for your reply.
It is not a site column, attached screenshot.
I have mapped on the tenant level.

Let me know if you need anything.
![doc lib](https://github.com/microsoft-search/pnp-modern-search/assets/46070806/673f29e1-90d0-4693-b71f-c1a7913b1b36)

**Reply by kasperbolarsen** [reply]
OK, well well.

I have read that it should be possible to use List Columns in search, however I don't recall the details, as I always use Content Types (very often from the Content type hub) and Site Columns. 
On top of that I have been burned before by using Lookup columns, so I very much prefer to use Managed Metadata (Term Store) instead. This approach also allows me to reuse the setup on multiple Site Collections/Portals :-)
I hope somebody else can help you with your current setup. Best of luck.

**Reply by wobba** [reply]
Haven't checked lookup columns in a while, so if one of you mappings work the others should as well. Ensure that the display name of the lookup column is what you want shown - as I believe it will pick up the display name used in the list. Also, have you re-indexed the data after the mapping was made?

**Reply by jerripothula** [reply]
Yes. I have re-indexed after the mapping.
see the below document and their refinable string values, it is showing for some and others not.
I still can't understand why it is picking up for some others not.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/46070806/427cffe2-7b27-4703-9977-11c2dabbd5cc)
![image](https://github.com/microsoft-search/pnp-modern-search/assets/46070806/f6082756-f652-4de9-be99-9871eac182ae)
![image](https://github.com/microsoft-search/pnp-modern-search/assets/46070806/2b704ee4-84a5-4803-a2c8-5b12b7b1895a)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/46070806/4e0e9c96-c34b-4040-a024-7e9bef2e0681)
![image](https://github.com/microsoft-search/pnp-modern-search/assets/46070806/4ba4f16e-8215-4187-bba3-1e303e2a3c7b)

**Reply by wobba** [reply]
Hard to tell indeed and you can try a support case with Microsoft on this inconsistent behavior. Also taking a look at the crawl log entries for the failing items is worth a try. Maybe it shows something.

**Comment by 0204xls** [comment]
If you're still having trouble, you might try mapping to a RefinableString**First** property. I use those for lookup columns in search and they work well.
